testing/src/main/java/net/consensys/linea/testing/ToyExecutionEnvironment.java

```diff
@@ -254,7 +254,7 @@ private MainnetTransactionProcessor getMainnetTransactionProcessor() {
         new MessageCallProcessor(evm, precompileContractRegistry);
 
     final ContractCreationProcessor contractCreationProcessor =
-        new ContractCreationProcessor(evm, false, List.of(), 0);
+        new ContractCreationProcessor(evm, false, List.of(), 1);
```

Comment:
The only question I have with respect to that is: when we query an account that is about to being deployed will we get a zero nonce ?

Reply:
We'll find out soon enough when connecting HUB constraints 😄